src/components/Member/MyPage/Cards/MyDogCard.tsx

```diff
@@ -132,14 +139,14 @@ const MyDogCard = ({
     >
       <DogDeleteButton
         isOpen={isOpen}
-        onClick={
-          dogLength <= 1 || enrolledDogs.length <= 1 ? openInvalidInputPopup : openDeleteDogPopup
-        }
+        onClick={validDogs.length <= 1 ? openInvalidInputPopup : openDeleteDogPopup}
```

Comment:
조건문을 작성할땐 부정조건문 보단 긍정조건문으로 작성하는게 가독성면에서 유리합니당~!

```suggestion
        onClick={validDogs.length > 1 ? openDeleteDogPopup : openInvalidInputPopup}
```

참고) https://latte1114.tistory.com/519

Reply:
좋은 정보 감사합니다! 바로 반영할게요 :)